### PR TITLE
Remove login_access_sockets

### PR DESCRIPTION
**Pull request by cmouse**
JIRA: DOV-5016

### CODE REVIEW

**Review by sirainen — DISMISSED** [review]

**Comment by sirainen** [comment]
Hm. Upgrading page update also?

**Comment by cmouse** [comment]
Merge #423 first

**Comment by sirainen** [comment]
https://github.com/dovecot/documentation/pull/423 is merged, but still no mention in upgrading page.

**Comment by sirainen** [comment]
https://github.com/dovecot/documentation/pull/423 is merged, but still no mention in upgrading page.

**Comment by cmouse** [comment]
Merge #437 first.

File: source/installation_guide/upgrading/from-2.3-to-3.0.rst [hunk 1]
```diff
@@ -78,3 +78,8 @@ Removed features and their replacements
 | dsync: Remove -D parameter                                 | Parameter for disabling mailbox rename syncing removed.                                  |
 |                                                            | It hasn't been necessary for a long time, and it is broke                                |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| setting                                                    | Use :ref:`authentication-lua_based_authentication` instead.                              |
+| ``login_access_sockets``                                   | Dovecot will fail to start if this setting is present in configuration.                  |
```
Comment (sirainen) [inline source/installation_guide/upgrading/from-2.3-to-3.0.rst:82]:
```
``login_access_sockets`` setting 
```
?..

**Review by sirainen — CHANGES_REQUESTED** [review]

**Review by sirainen — APPROVED** [review]

**Pull request merged**